sphinx_gallery/tests/test_binder.py

```diff
@@ -75,6 +76,18 @@ def test_binder():
         url = check_binder_conf(conf6)
     excinfo.match(r"`dependencies` value should be a list of strings")
 
+    # Missing requirements file should raise an error
+    with pytest.raises(ValueError) as excinfo:
```

Comment:
why not simpler `ValueError, match="Couldn't find the Binder requirements")`?

Also I'd just put the line that is expected to fail in the context handler, even though in principle it should be the last one anyway

Reply:
>Also I'd just put the line that is expected to fail in the context handler, even though in principle it should be the last one anyway

what does this mean exactly? :-)

Reply:
Answered in commit form

https://github.com/sphinx-gallery/sphinx-gallery/pull/439/commits/a69592a7c62d549aa63b9a2163c44651243e7826